### ISSUE TITLE
Undefinded method in  Mage_Customer_Model_Api2_Customer_Address

### ISSUE THREAD

**Issue by sreichel** [OpenMage/magento-lts#680]
Method `_getDefaultAddressesInfo()` is not definded in   `Mage_Customer_Model_Api2_Customer_Address`. It's set in `Mage_Customer_Model_Api2_Customer_Address_Rest` ...

```
    /**
     * Load customer address by id
     *
     * @param int $id
     * @return Mage_Customer_Model_Address
     */
    protected function _loadCustomerAddressById($id)
    {
        /* @var Mage_Customer_Model_Address $address */
        $address = Mage::getModel('customer/address')->load($id);

        if (!$address->getId()) {
            $this->_critical(self::RESOURCE_NOT_FOUND);
        }
        $address->addData($this->_getDefaultAddressesInfo($address));

        return $address;
    }
```

**Comment by mattdavenport** [comment]
An update here. I'm showing the method in the latest `1.9.4.x` branch:

```php
    /**
     * Get array with default addresses information if possible
     *
     * @param Mage_Customer_Model_Address $address
     * @return array
     */
    protected function _getDefaultAddressesInfo(Mage_Customer_Model_Address $address)
    {
        return array(
            'is_default_billing'  => (int)$this->_isDefaultBillingAddress($address),
            'is_default_shipping' => (int)$this->_isDefaultShippingAddress($address)
        );
     }
```

I believe this issue can be closed.

**Comment by ADDISON74** [comment]
I would like to close this issue. As I understand from the last post it was fixes in 1.9.4.x.
